dashboard/src/components/schedule/ScheduleView.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    class="flex gap-4 w-full my-6"
+    :class="{
+      'flex-col': view === 'vertical',
+      'overflow-x-scroll flex-row min-h-[800px]': view === 'horizontal',
+    }"
+  >
+    <div
+      v-for="(sessions, hall) in schedule"
+      :key="hall"
+      :class="{
+        'min-w-[720px] flex-shrink-0': view === 'horizontal',
+      }"
+    >
+      <SessionListHeader
```

Comment:
There are built-in HTML tags that handle this particular use case of collapsible components: [`details`, `summary`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). 

Styling can be done as per how we want. The main advantages are: 

1. We needn't monitor the state of the Vue components, as it's builtin with the tag. 
2. It enhances accessibility - as it's detected well by screen-readers etc.

Which makes it relevant as we are trying to make IndiaFOSS accessible as mush as possible. 

This could also be taken as a potential refactor in future.

Reply:
Tried experimenting with it as whole with 0 style add-ons: 

```
<details>
        <summary>
          {{ hall }}
          <!-- <SessionListHeader
          :title="hall"
          :collapsible="isCollapsible"
          @collapse-hall="toggleCollapse(hall)"
          /> -->
        </summary>

        <SessionList
          v-if="!isCollapsed(hall)"
          :sessions="sessions"
          :view="view"
        />
</details>
```

Result: 
![Screenshot 2024-08-29 at 11 45 49 PM](https://github.com/user-attachments/assets/eedf0b81-8098-42a2-ac89-7009bc7103f9)

Reply:
The Schedule List header is collapsible sometimes, and other times its not. Depends on the view. Can try experimenting this in future

Reply:
> header is collapsible sometimes, and other times its not

This can be handled via tag attributes - check documentation for more details. This needn't be implemented now also, as I've mentioned as: 

> This could also be taken as a potential refactor in future.